build.gradle

```diff
@@ -151,7 +151,12 @@ project(':cruise-control-core') {
   }
 
   dependencies {
+    configurations.all {
```

Comment:
if this project had proper modules instead of lumping them all to a single gradle file you could have avoided repeating this 2 times below - just sayin'

Reply:
I agree, we should fix that -- just created an issue about this: https://github.com/linkedin/cruise-control/issues/1478.